apiary.apib

```diff
@@ -0,0 +1,347 @@
+                {
+                    "type": "roles",
+                    "name": "Stack Manager",
+                    "users": {
+                        "count": 2,
+                        "data": [
+                            {
+                                "type": "users",
+                                "id": "2379",
+                                "name": "Albus Dumbledore",
+                                "email": "sherbert.lemon@hogwarts.uk",
+                            },
+                            {
+                                "type": "users",
+                                "id": "21279",
+                                "name": "Severus Snape",
+                                "email": "halfbloodprince@hogwarts.uk",
+                            }
+                        ]
+                    }
+                },
+                {
+                    "type": "roles",
+                    "name": "Deployment Manager",
+                    "users": {
+                        "count": 1,
+                        "data": [
+                            {
+                                "type": "users",
+                                "id": "224",
+                                "name": "Minerva McGonagall",
+                                "email": "mcgryffindor@hogwarts.uk",
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+
+
+# Snapshots [/snapshots]
+
+## List all snapshots [GET /naut/project/$name/snapshots]
+
+Returns a list of snapshots associated with a stack
+
++ Response 200 (application/json)
+
+        {
+            "count": 1,
+            "data": [
+                {
+                    "type": "snapshots",
+                    "id": "398433",
+                    "created": "25-01-2017",
+                    "author": {
+                        "data": [
+                            {
+                                "type": "users",
+                                "id": "2379",
+                                "name": "Albus Dumbledore",
+                                "email": "sherbert.lemon@hogwarts.uk",
+                            }
+                        ]
+                    },
+                    "source": {
+                        "data": [
+                            {
+                                "type": "environments",
+                                "name": "prod"
+                            }
+                        ]
+                    },
+                    "owner":  {
+                        "data": [
+                            {
+                                "type": "environments",
+                                "name": "uat"
+                            }
+                        ]
+                    },
+                    "mode": "db",
+                    "size": "2096",
+                    "download_link": "deploy.cwp.govt.nz/path/to/snapshot.sspak"
```

Comment:
This is yet to be worked out. It will in theory have no download_link and no size - but we can probably be more explicit